Yeast_Project/Scripts/2_Node_Summary_Histo.ipynb

```diff
@@ -0,0 +1,222 @@
+      "text/plain": [
+       "<Figure size 1000x600 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Plot a histogram for the \"Number of nodes\" column using Matplotlib\n",
+    "plt.figure(figsize=(10, 6))\n",
+    "plt.hist(df['Nodes in prize'], bins=30, color='blue', edgecolor='black', alpha=0.7)\n",
+    "plt.title('Histogram of Nodes in prize')\n",
+    "plt.xlabel('Nodes in prize')\n",
+    "plt.ylabel('Frequency')\n",
+    "plt.show()\n",
+    "\n",
+    "# Alternatively, using Seaborn for a more styled plot\n",
+    "plt.figure(figsize=(10, 6))\n",
+    "sns.histplot(df['Number of nodes'], bins=30, kde=True, color='blue')\n",
```

Comment:
```suggestion
    "sns.histplot(df['Number of nodes'], bins=30, color='blue')\n",
```
I don't think the kernel smoothing is meaningful for this histogram, we can turn it off

Reply:
Turned off the smoothing.